phpunit.xml

```diff
@@ -1,7 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.1/phpunit.xsd" bootstrap="vendor/autoload.php"
-         colors="true" cacheResult="false" cacheDirectory=".phpunit.cache">
+<phpunit
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.1/phpunit.xsd"
+    bootstrap="vendor/autoload.php"
+    colors="true"
+    cacheResult="false"
+    cacheDirectory=".phpunit.cache"
+>
```

Comment:
PHPUnit encourages developers to be 🎨 artistic!

Reply:
This is 🎈  _Windows Safe Mode_.
`xmllint` produces proper XML files for machines: the `phpunit` element will on one line.